packages/arb-token-bridge-ui/src/hooks/__tests__/useNetworks.test.ts

```diff
@@ -232,4 +233,31 @@ describe('sanitizeQueryParams', () => {
       })
     })
   })
+  describe('when `token=eth` is defined', () => {
```

Comment:
you added the tests under `sanitizeQueryParams`, i think it should be under a `describe` of `sanitizeTokenQueryParam`